### ISSUE TITLE
Doesn't seem to work any longer.

### ISSUE THREAD

**Issue by jesstesla** [philipbeel/Tweetable#33]
It looks as though this no longer works. I believe the Get My Tweets api has changed.

**Comment by voyageravv** [comment]
If someone wants get work tweetable, i give you the index.php to replace remote API

CONSUMER y access Tokens in:

https://apps.twitter.com/

Abraham  twitter Oauth in...

https://twitteroauth.com

<?php
require "twitter/autoload.php";
$CONSUMER_KEY='Your Consumer key';
$CONSUMER_SECRET='Your Consumer Secret';
$accessToken='Your Access Token';
$accessTokenSecret='Your Access Token Secret';
$id=$_GET['screen_name'];
$limit=$_GET['limit'];
use Abraham\TwitterOAuth\TwitterOAuth;
$connection = new TwitterOAuth($CONSUMER_KEY, $CONSUMER_SECRET, $accessToken, $accessTokenSecret);
$twitterData = $connection->get(
          'statuses/home_timeline',
          array(
              'screen_name'     => $id,
              'count'           => $limit
          )
      );
$tweets=array();
$i=0;
 if(!empty($twitterData)){
  foreach($twitterData as $tweet){
    if (!empty($tweet->id))
    {
      $tweets[$i]['tweet_date']=date("YM d H:i:sTZY",strtotime($tweet->created_at));
      $tweets[$i]['response']=$tweet->text;
      $i++;
    }
  }
  }
$result['tweets']=$tweets;
echo json_encode($result);
?>

**Comment by philipbeel** [comment]
Thanks for this @voyageravv If you would like to raise pull request that would be great. Alternatively I will review this and integrate it myself.

**Comment by voyageravv** [comment]
Yes, if you integrate it that would be great... I didn't pull request, because I don know if it's possible to include third party code (Abraham library). If you could do it... It's perfect.
